backend/db/refresh.py

```diff
@@ -57,6 +68,7 @@ def refresh_db(
     if is_refresh_active():
         print('INFO: Refresh already in progress. When that process completes, it will restart again. Exiting.')
         update_db_status_var('new_request_while_refreshing', start_time, local)
+        trigger_resolve_failures(resolve_fetch_failures_excess_items, resolve_fetch_failures_0_members, local)
```

Comment:
## Always run fetch failures if needed
Bug fix: Now, if the refresh doesn't need to run, it still triggers resolution of fetch failures.

This fix is good because when 0 members occur because of a new draft, the members might not be ready by the end of the refresh, and it'd be good if the next time the refresh ran, it tried to pick them up.